simple/stats/constants.py

```diff
@@ -22,6 +22,7 @@
 CONFIG_JSON_FILE_NAME = "config.json"
 REPORT_JSON_FILE_NAME = "report.json"
 TRIPLES_FILE_NAME = "triples.csv"
+EMBEDDINGS_FILE_NAME = "embeddings.csv"
```

Comment:
Might be worth versioning it

Reply:
Maybe add TODO

Reply:
Our current thinking is that the user (the mixer loader primarily) specifies separate output directories on each run (so that each run can be tracked separately) which implicitly acts as a versioning mechanism.

Would be good to get your thoughts on this as well.